index.md

```diff
@@ -75,6 +75,10 @@ In your report please include:
 
 Reports will receive urgent and immediate attention from the Code of Conduct Committee. Please refer to the [Enforcement Manual][] for the complete information on how reports will be processed.
 
+### Confidentiality
+
+The mailing list is addressed to members of [Code of Conduct Committee][], and reports will only be shared with others outside the Committee with permission of the reporter. No identity will be made public beyond the Code of Conduct Committee and the ECMA Executive Committee unless the individuals concerned have agreed to it. If you would prefer to share your report with only certain members of the Code of Conduct Committee, you may email it to a subset using their listed email addresses; this subset will not share the report with any others outside the subset without permission of the reporter.
```

Comment:
You could consider adding a note that it is helpful to know a person's identity, but that the code of conduct committee will consider anonymous reports. You could also suggest two avenues for submitting such an anonymous report - a throwaway email address, or onion routing via trusted intermediaries.

Reply:
Done